folly/lang/Bits.h

```diff
@@ -68,6 +68,7 @@ namespace folly {
 
 #if __cpp_lib_bit_cast
 
+#include <bit>
```

Comment:
This will work for MSVC - we define all feature-test macros when any header is included - but it's not portable. For a portable fix the code needs to include `<bit>` or `<version>` before checking `__cpp_lib_bit_cast`. I suggest:
```c++
#ifdef __has_include
#if __has_include(<bit>)
#include <bit>
#endif
#endif

#if __cpp_lib_bit_cast
using std::bit_cast
#else
// ...
```

Reply:
I know, I was trying to be minimal